models/event.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const Build = require('./build');
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+const Pipeline = require('./pipeline');
+const Scm = require('../core/scm');
+const Workflow = require('../config/workflow');
+
+const buildId = Joi.reach(Build.base, 'id');
+
+const MODEL = {
+    id: Joi
+        .string().hex().length(40)
+        .description('Identifier of this build')
+        .example('4b8d9b530d2e5e297b4f470d5b0a6e1310d29c5e'),
+    builds: Joi
```

Comment:
I assume we're going to move away from `/jobs/:jobId/builds` and into `/events/:eventId/builds`  If that's the case, wouldn't the UI be making less calls?

 - `/pipelines/:pipelineId/events` - list of events
 - `/events/:eventId/builds` - list of builds for that event